### DISCUSSION TITLE
Ideas Taken Here

### DISCUSSION THREAD

**Discussion by morpheuslord** [morpheuslord/GPT_Vuln-analyzer#4] [Ideas]
If you have any great **ideas** on what we can add to this already existing code or on how I can improve this code please feel free to share.

**Comment by drg3nz0** [comment]
I tried your project, and it's great! I have a suggestion: consider adding voice commands in the future. Many people use voice search nowadays, and although it might not be necessary, it could be a nice feature. Keep up the good work!

**Reply by morpheuslord** [reply]
Nice Idea but currently such a feature requires possibly rewriting the entire main driver code so this might be my next work in progress for the next release. Thanks for the idea and do suggest any other if you have or you might want to see implemented.
